dts/arm/cypress/psoc6.dtsi

```diff
@@ -81,6 +81,17 @@
 			status = "disabled";
 			label = "uart_6";
 		};
+
+		psoc6_ipm7: mailbox@402300E0 {
```

Comment:
The hex constant needs to be in lower-case, e.g. `mailbox@402300e0`. It is generating a warning right now.